python/cuml/cuml/tests/test_array.py

```diff
@@ -728,3 +728,20 @@ def test_order_to_strides(input_type, dtype, shape, order):
         np.array(_order_to_strides(order, shape, dtype))
         == np.array(input_array.strides)
     )
+
+
+@pytest.mark.parametrize("kind", ["dataframe", "series"])
+def test_output_pandas(kind):
```

Comment:
There are existing tests in this file that check our output handling, but they all are run using `hypothesis` which isn't run on our `cudf.pandas` integration tests. Another case for doing something about #6380.